opam

```diff
@@ -14,11 +14,14 @@ depends: [
   "ocamlfind"
   "lwt" {>= "2.4.3"}
   "cstruct" {>= "1.9"}
+  "mirage-block"
   "mirage-types-lwt" {>= "3.0.0"}
   "ipaddr"
   "io-page"
```

Comment:
You also need "io-page-unix"

Reply:
Hm, why `-unix`?

Reply:
Hmm, it seems that `Executable parse_test` in the `_oasis` file requires `io-page.unix`. In the jbuilder port in a followup PR we can make this a `{test}` dependency.

Reply:
Ah, I see.  As long as it's marked `{test}` it's fine.

Reply:
Oh I see, without additional changes to the `_oasis` configuration we cannot currently flag it as test. Imho is fine to leave it as is in this PR, and make it `{test}` in the `jbuilder` port. What do you think?